clubs/README.md

```diff
@@ -50,66 +49,64 @@ _(each of the sections below corresponds with a bullet point)_
-leader.
+leader who delegates authority.
 
 ### Teacher Sponsor and Space
 
 A good teacher sponsor is key to having access to regular space for club
-meetings and a good relationship with your school. Your teacher sponsor is the
-person who will regularly lend you a classroom for after-school meetings. And
-when things go wrong with the school administration, your teacher sponsor will
-be your advocate. Choosing the wrong teacher sponsor can be fatal to clubs, so
-give it careful thought.
-
-The best teacher sponsors are personally invested in spreading coding to
-students. This is generally the teacher most likely to start a computer science
-class (or, if you already have one, the current computer science teacher). If
-you're new to your school and not very familiar with the teachers, a good place
-to start looking is the math and science teachers. If your school has a computer
-lab, start with the teachers that teach classes in it (or if your school has
+meetings and a good relationship with your school. Your sponsor is the person
+who will lend you a classroom for after-school meetings. When things go wrong
+with the school administration, your er sponsor will be your advocate. Choosing
+the wrong sponsor can be fatal to clubs, so give it careful thought.
+
+The best sponsors are personally invested in spreading coding to students. This
+is generally the teacher most likely to start a computer science class (or, if
+you already have one, the current computer science teacher). If you're new to
+your school and not very familiar with the teachers, a good place to start
+looking is the math and science department. If your school has a computer lab,
+start with the teachers that teach classes in it (or if your school has
 classrooms with computers in it, start with the teachers with computer
 classrooms).
 
-Once you've decided on a few potential teacher sponsors, it's time to reach out
-to them and tell them about your idea to start a Hack Club. The best way to do
-this is to come to their classroom right after school ends (or during lunch) and
-talk to them in person. If you don't already know the teacher, introduce
-yourself, tell them that you're looking for a teacher sponsor for this thing
-called a Hack Club you're starting, and explain the club to them. Here's an
-example of what you might want to start off by saying:
+Once you've decided on a few potential teacher sponsors, reach out to them and
+tell them about your idea to start a Hack Club. The best way to do this is to
+come to their classroom right after school ends (or during lunch) and talk to
+them in person. If you don't already know the teacher, introduce yourself, tell
+them that you're looking for a teacher sponsor for this thing called a Hack Club
+that you're starting, and explain the club to them. Here's an example of what
+you might want to say:
 
 > Hi Ms. Byrd, I'm Zach. I'm currently a freshman and I'm starting a Hack Club
```

Comment:
I never actually had her as a teacher, but she's the first that came to
mind :-P.

On Sun, Jan 17, 2016 at 2:04 AM, Max Wofford <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/658#discussion_r49944040>:
>
> >  classrooms with computers in it, start with the teachers with computer
> >  classrooms).
> >
> > -Once you've decided on a few potential teacher sponsors, it's time to reach out
> > -to them and tell them about your idea to start a Hack Club. The best way to do
> > -this is to come to their classroom right after school ends (or during lunch) and
> > -talk to them in person. If you don't already know the teacher, introduce
> > -yourself, tell them that you're looking for a teacher sponsor for this thing
> > -called a Hack Club you're starting, and explain the club to them. Here's an
> > -example of what you might want to start off by saying:
> > +Once you've decided on a few potential teacher sponsors, reach out to them and
> > +tell them about your idea to start a Hack Club. The best way to do this is to
> > +come to their classroom right after school ends (or during lunch) and talk to
> > +them in person. If you don't already know the teacher, introduce yourself, tell
> > +them that you're looking for a teacher sponsor for this thing called a Hack Club
> > +that you're starting, and explain the club to them. Here's an example of what
> > +you might want to say:
> >
> >  > Hi Ms. Byrd, I'm Zach. I'm currently a freshman and I'm starting a Hack Club
>
> "Ms. Byrd" lul
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/658/files#r49944040>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta